icu4c/source/i18n/number_output.cpp

```diff
@@ -0,0 +1,156 @@
+// © 2019 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#include "unicode/utypes.h"
+
+#if !UCONFIG_NO_FORMATTING
+
+#include "unicode/numberformatter.h"
+#include "number_utypes.h"
+#include "util.h"
+#include "number_decimalquantity.h"
+
+U_NAMESPACE_BEGIN
+namespace number {
+
+
+FormattedNumber::FormattedNumber(FormattedNumber&& src) U_NOEXCEPT
```

Comment:
optional: why not call this file formattednumber.cpp ?

Reply:
I name the files with the `number_` prefix.

- `number_formatted` was too close to `number_formatimpl`
- `number_formattednumber` was a bit long

so I went with `number_output` for lack of a better option.